### PR TITLE
Remove UpdateDeploymentChain function from deploymentchain store interface

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Follow PR #3207

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/datastore/deploymentchainstore.go [hunk 1]
```diff
@@ -183,3 +184,13 @@ func (s *deploymentChainStore) UpdateDeploymentChain(ctx context.Context, id str
 		return dc.Validate()
 	})
 }
+
+func (s *deploymentChainStore) AddDeploymentToBlock(ctx context.Context, chainID string, deployment *model.Deployment) error {
```
Comment (nghialv) [inline pkg/datastore/deploymentchainstore.go:188]: How about `UpdateNodeDeployment`

Comment (khanhtc1202) [reply pkg/datastore/deploymentchainstore.go:188]: Update is a bit hard to get what it is, lets me rename it as `AddNodeDeployment`

Comment (nghialv) [reply pkg/datastore/deploymentchainstore.go]: Sure.

**Comment by knanao** [comment]
Here you go!
/approve

**Pull request merged**